### PR TITLE
MINOR: Remove Dead Code in Routing

### PR DESCRIPTION
**Pull request by original-brownbear**

### CODE REVIEW

**Review by cbuescher — COMMENTED** [review]
@original-brownbear nice stats, LGTM except for one nit where we might want to change the method name to reflect its current use better. Wdyt?

File: server/src/main/java/org/elasticsearch/cluster/routing/allocation/DiskThresholdSettings.java [hunk 1]
```diff
@@ -232,7 +231,6 @@ private void setHighWatermark(String highWatermark) {
 
     private void setFloodStageRaw(String floodStageRaw) {
```
[the remark this answers is not in the document]
Comment (original-brownbear) [reply server/src/main/java/org/elasticsearch/cluster/routing/allocation/DiskThresholdSettings.java]: @cbuescher makes sense, just drop the `raw` from the name maybe ? :)

[the remark this answers is not in the document]
Comment (cbuescher) [reply server/src/main/java/org/elasticsearch/cluster/routing/allocation/DiskThresholdSettings.java]: Sounds good. Also I didn't see this was a private method, so naming doesn't matter that much IMHO, but still nice if it is changed.

**Review by cbuescher — APPROVED** [review]
LGTM (mod the discussed naming change, but that doesn't require an additional review I think)

**Comment by original-brownbear** [comment]
@cbuescher thanks!

**Pull request merged**